pkg/bgp/speaker/events.go

```diff
@@ -35,6 +36,10 @@ type nodeEvent struct {
 	// (read comparable) in Go.
 	labels   *map[string]string
 	podCIDRs *[]string
+	// withDraw will be set when a Delete node event occurs.
+	// the reduction of this event will elicit a withdrawal
+	// of all bgp routes.
+	withDraw *bool
```

Comment:
This doesn't need to be a pointer because `bool` is hashable. Making these fields pointers is only required for types that are not hashable such as maps or slices. 

I guess 
>// The following fields must be a pointers because they are not hashable
	// (read comparable) in Go.

was a bit misleading since it can be interpreted as all fields? Probably worth clarifying it.

Reply:
You are correct. 

I'm not exactly aware of why this consideration is here in the first place? I didn't see any code doing comparisons of these event types and fields. Can you clue me in?

Reply:
The issue is that the event queuing uses a `map` under the hood, I believe. So if you were to remove all the pointers from the `nodeEvent`, you'll get a panic saying "unhashable type" or something like that. https://blog.golang.org/maps#TOC_5.

Reply:
got it. It's required by the k8s workqueue.